README.md

```diff
@@ -212,13 +213,15 @@ Do you have questions? Feel free to [request an invitation](https://surveys.jetb
 
 ## Pull requests
 
-We actively welcome your pull requests. However, linking your work to an existing issue is preferred.
+We actively welcome your pull requests. However, linking your work to an [existing issue](https://youtrack.jetbrains.com/issues/EXPOSED) is preferred.
```

Comment:
@vnikolova  It seems like this section has been mostly left alone, while the [website Contributing page](http://jetbrains.github.io/Exposed/contributing.html) has been more thoroughly updated.
I've added some details here but it is still not on the same level as the content in `Contributing.md`. Personally, I would remove this section entirely, and rely on the 2 existing links that lead to the guidelines (1 in 'Links' above and another in 'Contributing' below).
I would even consider moving the 'Contributing' section above the 'Examples' section so it's more visible.
Please let me know what you think should be done with this section.

Reply:
✅